cython_lint/cython_lint.py

```diff
@@ -836,6 +843,39 @@ def traverse(tree: ModuleNode) -> Iterator[NodeParent]:
         yield node_parent
 
 
+def _get_config(paths: list[pathlib.Path]) -> dict[str, Any]:
+    """Get the configuration from a config file
+
+    Search for a pyproject.toml or a setup.cfg file in common
+    parent directories of the given list of paths.
+    """
+    paths = [path.resolve() for path in paths]
```

Comment:
aren't we iterating over `paths` multiple times like this? can we just call `resolve()` directly one level up?

Reply:
yep, done